src/adapters/conversant.js

```diff
@@ -0,0 +1,223 @@
+'use strict';
+var VERSION = '2.0.1',
+    CONSTANTS = require('../constants.json'),
+    utils = require('../utils.js'),
+    bidfactory = require('../bidfactory.js'),
+    bidmanager = require('../bidmanager.js'),
+    adloader = require('../adloader');
+
+/**
+ * Adapter for requesting bids from Conversant
+ */
+var ConversantAdapter = function () {
+  var w = window,
+    n = navigator;
+
+  // production endpoint
+  var conversantUrl = '//media.msg.dotomi.com/s2s/header?callback=$$PREBID_GLOBAL$$.conversantResponse';
```

Comment:
Do you know what ad sizes are requested? We're currently working on a fix for the headers of our 204 responses